doc/source/contributing.rst

```diff
@@ -287,6 +286,45 @@ is usually quite straightforward.
 ------------------------------
 
 Once you've made changes and pushed them to your forked repository, you then
-submit a pull request to have them integrated into the *geopandas* code base.
+submit a pull request to have them integrated into the *GeoPandas* code base.
 
 You can find a pull request (or PR) tutorial in the `GitHub's Help Docs <https://help.github.com/articles/using-pull-requests/>`_.
+
+Style Guide & Linting
```

Comment:
ok ive added the precommit / style guide linting info here. see what you both think!!

Reply:
Thanks, that looks good!